azure-pipelines.yml

```diff
@@ -31,6 +32,11 @@ jobs:
       go.list.filter: '| grep -v vendor'
 
     steps:
+    - task: GoTool@0
+      inputs:
+        version: '$(go.version)'
+      displayName: "Select Go Version"
+
```

Comment:
In `create go workspace` there is a LoC that prepends `GOROOT/bin` to the path. Since this is already executed by the GoTool, I'm going to remove it after I get a green run.

![image](https://user-images.githubusercontent.com/45376673/75595277-96a4a200-5a40-11ea-8299-40d7ae6800f8.png)